bitnami/cilium/templates/envoy/metrics-service.yaml

```diff
@@ -22,10 +22,10 @@ spec:
   ports:
     - name: tcp-metrics
       port: {{ .Values.envoy.metrics.service.port }}
-      targetPort: metrics
+      targetPort: {{ ternary "metrics" "envoy-metrics" .Values.envoy.useDaemonSet }}
       protocol: TCP
   {{- $podLabels := include "common.tplvalues.merge" (dict "values" (list .Values.envoy.podLabels .Values.commonLabels) "context" .) | fromYaml }}
   selector: {{- include "common.labels.matchLabels" ( dict "customLabels" $podLabels "context" $ ) | nindent 4 }}
     app.kubernetes.io/part-of: cilium
-    app.kubernetes.io/component: envoy
+    app.kubernetes.io/component: {{ ternary "envoy" "agent" .Values.envoy.useDaemonSet }}
```

Comment:
Not sure if I understand this ternary, isn't this part of the envoy component?

Reply:
Yes, but envoy can be run independently (as a daemonset) or a sidecar as part of the agent